Misc/Positions_Area Positions 4 External.txt

```diff
@@ -52,8 +52,10 @@ LFEE_CTR:Reims Control:128.300:EE:C:LFEE:CTR:LFEE:CTR:5700:5767
 LFRR_CTR:Brest Control:125.500:BR:C:LFRR:CTR:LFRR:CTR:5640:5657
 LFRR_E_CTR:Brest Control:136.000:BRE:C:LFRR:CTR:LFRR:CTR:5640:5657
 LFRR_J_CTR:Brest Control:132.500:BRJ:C:LFRR:CTR:LFRR:CTR:5640:5657
-LFRR_Q_CTR:Brest Control:130.220:BRQ:C:LFRR:CTR:LFRR:CTR:5640:5657
-LFRR_V_CTR:Brest Control:124.770:BRV:C:LFRR:CTR:LFRR:CTR:5640:5657
+LFRR_UE_CTR:Brest Control:130.220:BRQ:C:LFRR:CTR:LFRR:CTR:5640:5657
+LFRR-UE_CTR:Brest Control:130.220:BRQ:C:LFRR:CTR:LFRR:CTR:5640:5657
+LFRR_VK_CTR:Brest Control:124.770:BRV:C:LFRR:CTR:LFRR:CTR:5640:5657
+LFRR-VK_CTR:Brest Control:124.770:BRV:C:LFRR:CTR:LFRR:CTR:5640:5657
```

Comment:
```suggestion
LFRR-VK_CTR:Brest Control:124.770:BRV:C:LFRR-VK:CTR:LFRR:CTR:5640:5657
```

Reply:
@ChrisXPP for the sake of simplicity I think we'll keep them as 2 PRs

Reply:
@GeekPro101 But then please with correct frequencies (130.225 and 124.775 instead of 130.220 and 124.770)...

Reply:
@GeekPro101 I added a new commit containing frequencies ending with .x25/.x75.